smf/clickhouse/manifest.xml

```diff
@@ -16,13 +17,17 @@
       <envvar name="LD_LIBRARY_PATH" value="/opt/ooce/pgsql-13/lib/amd64" />
```

Comment:
Is this load bearing?!

Reply:
I don't *think* so? I thought it was only actually necessary for the Nexus zone, since **that** one links against postgres.

Reply:
I went ahead and removed this for both Clickhouse and Cockroachdb